_summaries/chen2020learning_2.md

```diff
@@ -1,19 +1,61 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
+title: Learning Continuous Image Representation with Local Implicit Image Function
+paper: chen2020learning
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: # SShivang
+score: 8/10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+What is the core idea?
+
+Real world visual input is continuous in nature, however images are represented as 2D arrays of pixels. Precision and complexity is controlled by the resolution of the image. However, for image models, images have to be resized to the same size (thus losing information about the resolution and losing fidelity to the original image). By modeling the image with the function, Local Implicit Image Function (LIIF), the authors show that they can generate an image in an arbitrary resolution.
+
+![Example Input and Outputs](chen2020learning_2_1.png)
+
+How is it realized (technically)?
+
+![Function Definition](chen2020learning_2_2.png)
+
+The goal of the model is to take a 2D coordinate x, the vector z and return the signal, which is a RGB value. Unlike implicit neural representations, where the output function is learned for each instance, the LIIF model is learned and generalized to all instances.
+
+![Example z and x values](chen2020learning_2_4.png)
+![Deriving the image given model](chen2020learning_2_3.png)
+
+This figure, shows how the model works intuitively. We divide the image into a grid and for each grid cell we calculate a latent vector representation z. We use the vector z, associated with the grid square closest to the coordinate.
+
+![Example z and x values](chen2020learning_2_5.png)
+![Example z and x values](chen2020learning_2_6.png)
+
+The paper also makes certain optimizations to this original idea. For each of the latent vectors, they concatentate the neighboring cell latent vector to find the new latent vector. (Feature Unfolding)  This allows for a richer representation. Another optimization they make is that instead of using the closest latent vector they use the weighted average of latent vector (weighted by the distance from the coordinate x). (Local Ensemble)
```

Comment:
Maybe you could state why they do these optimizations. The Local ensemble optimization is applied to remove discontinuities that arise when using the originally proposed function.